functions.js

```diff
@@ -34,10 +37,10 @@ function createResponseObject(resolve, reject) {
     error: function(error) {
       reject(new Parse.Error(Parse.Error.SCRIPT_FAILED, error));
     }
-  }
+  };
 }
 
-router.route('POST', '/functions/:functionName', handleCloudFunction);
+router.route('POST', '/functions/:functionName+', handleCloudFunction);
```

Comment:
+ will allow to have a structured calls on functions, such as
```
Parse.Cloud.run('workflow/list', { objs : true}), 
```
extending functions reach.
It is an optional route parameter from express, it will be ignored if it is not present and it will work as pervious.